components/director/pkg/customerrors/error_codes.go

```diff
@@ -0,0 +1,13 @@
+package customerrors
+
+type ErrorCode int
+
+const (
+	UnhandledError ErrorCode = iota
+	InternalError
+	NotFound
+	NotUnique
+	TenantNotFound
```

Comment:
Please, add a `TenantNotExisting` error. It happens when user passes a tenant, that doesn't have a mapping in Director